src/Reflection/BetterReflection/SourceLocator/PhpFileCleaner.php

```diff
@@ -1,293 +0,0 @@
-<?php declare(strict_types = 1);
-
-namespace PHPStan\Reflection\BetterReflection\SourceLocator;
-
-use function array_keys;
-use function implode;
-use function preg_match;
-use function preg_quote;
-use function strlen;
-use function substr;
-
-/**
- * @author Jordi Boggiano <j.boggiano@seld.be>
- * @see https://github.com/composer/composer/pull/10107
- */
-class PhpFileCleaner
```

Comment:
I remember seeing blackfire profiles where the PhpFileCleaner logic was on the hot path.

really like seeing this file beeing deleted :)